bot.py

```diff
@@ -44,25 +44,28 @@ async def store_message(connection, msg, generated_verse):
         msg.author.name, 
         msg.content, 
         msg.timestamp, 
-        msg.channel.name, 
+        msg.channel.name,
+        msg.tags['color'],
         generated_verse
     )
 
 class Bot(commands.Bot):
     def __init__(self):
         super().__init__(
             # Put your OAuth Password Token here. You can obtain one in https://twitchapps.com/tmi/
-            token="", 
+            token="oauth:jybeok26b7qi2in16uha11n34xqrf6", 
```

Comment:
leaked ;p

Reply:
erase the token, destroy it on twitch and create a new one. do it properly now so it dont get leaked again